app/services/container_deployment_service.rb

```diff
@@ -54,6 +54,6 @@ def optional_vms(vms)
   end
 
   def optional_deployment_types
-    %w(openshiftOrigin, openshiftEnterprise)
+    %w(openshiftOrigin openshiftEnterprise)
```

Comment:
@dkorn do you think we can maintain the camelcasing we regularly use in ManageIQ? (OpenShiftOrigin OpenShiftEnterprose)